dbt/models/model/schema.yml

```diff
@@ -55,20 +55,541 @@ sources:
+        description: '{{ doc("column_lake_michigan_dist_ft") }}'
+      - &prox_nearest_bike_trail_dist_ft
+        name: prox_nearest_bike_trail_dist_ft
+        description: '{{ doc("column_nearest_bike_trail_dist_ft") }}'
+      - &prox_nearest_cemetery_dist_ft
+        name: prox_nearest_cemetery_dist_ft
+        description: '{{ doc("column_nearest_cemetery_dist_ft") }}'
+      - &prox_nearest_cta_route_dist_ft
+        name: prox_nearest_cta_route_dist_ft
+        description: '{{ doc("column_nearest_cta_route_dist_ft") }}'
+      - &prox_nearest_cta_stop_dist_ft
+        name: prox_nearest_cta_stop_dist_ft
+        description: '{{ doc("column_nearest_cta_stop_dist_ft") }}'
+      - &prox_nearest_golf_course_dist_ft
+        name: prox_nearest_golf_course_dist_ft
+        description: '{{ doc("column_nearest_golf_course_dist_ft") }}'
+      - &prox_nearest_hospital_dist_ft
+        name: prox_nearest_hospital_dist_ft
+        description: '{{ doc("column_nearest_hospital_dist_ft") }}'
+      - &prox_nearest_major_road_dist_ft
+        name: prox_nearest_major_road_dist_ft
+        description: '{{ doc("column_nearest_major_road_dist_ft") }}'
+      - &prox_nearest_metra_route_dist_ft
+        name: prox_nearest_metra_route_dist_ft
+        description: '{{ doc("column_nearest_metra_route_dist_ft") }}'
+      - &prox_nearest_metra_stop_dist_ft
+        name: prox_nearest_metra_stop_dist_ft
+        description: '{{ doc("column_nearest_metra_stop_dist_ft") }}'
+      - &prox_nearest_park_dist_ft
+        name: prox_nearest_park_dist_ft
+        description: '{{ doc("column_nearest_park_dist_ft") }}'
+      - &prox_nearest_railroad_dist_ft
+        name: prox_nearest_railroad_dist_ft
+        description: '{{ doc("column_nearest_railroad_dist_ft") }}'
+      - &prox_nearest_water_dist_ft
+        name: prox_nearest_water_dist_ft
+        description: '{{ doc("column_nearest_water_dist_ft") }}'
+      - &prox_num_bus_stop_in_half_mile
+        name: prox_num_bus_stop_in_half_mile
+        description: '{{ doc("column_num_bus_stop_in_half_mile") }}'
+      - &prox_num_foreclosure_per_1000_pin_past_5_years
+        name: prox_num_foreclosure_per_1000_pin_past_5_years
+        description: '{{ doc("column_num_foreclosure_per_1000_pin_past_5_years") }}'
+      - &prox_num_pin_in_half_mile
+        name: prox_num_pin_in_half_mile
+        description: '{{ doc("column_num_pin_in_half_mile") }}'
+      - &prox_num_school_in_half_mile
+        name: prox_num_school_in_half_mile
+        description: '{{ doc("column_num_school_in_half_mile") }}'
+      - &prox_num_school_with_rating_in_half_mile
+        name: prox_num_school_with_rating_in_half_mile
+        description: '{{ doc("column_num_school_with_rating_in_half_mile") }}'
+
   - name: model.vw_card_res_input
     config:
       tags:
         - type_res
 
     columns:
-      - name: char_bldg_sf_95_percentile
+      # Columns inherited from vw_pin_shared_input
```

Comment:
Unfortunately, there d[oesn't seem to be a great way](https://stackoverflow.com/questions/24090177/how-to-merge-yaml-arrays/30770740#30770740) to merge YAML arrays using only built-in language features, hence this very messy anchor usage.

I suspect the only way around this would be to do some kind of pre-processing on the YAML schema files, but I'm not sure if it's worth doing. @jeancochrane, what do you think?

Reply:
**[Praise]** Yeah, to the best of my knowledge, anchors are the best way of performing this kind of reference. I think your instinct is right; the duplication is a bit annoying, but I think any type of preprocessing would introduce a huge layer of complexity that I don't think is worth it unless/until we can identify something that's truly prohibitive without it.